modules/core/src/main/java/org/apache/ignite/internal/processors/query/schema/SchemaIndexCachePartitionWorker.java

```diff
@@ -227,7 +237,7 @@ private void processKey(KeyCacheObject key) throws IgniteCheckedException {
                 GridCacheEntryEx entry = cctx.cache().entryEx(key);
 
                 try {
-                    entry.updateIndex(rowFilter, clo);
+                    entry.updateIndex(rowFilter, clo, indexCacheStat);
```

Comment:
Looks like all of these 3 params can be squashed into single closure right here.